src/Ocean/HydrostaticBoussinesq/HydrostaticBoussinesqModel.jl

```diff
@@ -1,7 +1,7 @@
 module HydrostaticBoussinesq
 
 export HydrostaticBoussinesqModel, HydrostaticBoussinesqProblem, OceanDGModel,
-       LinearHBModel
+       LinearHBModel, calculate_dt
```

Comment:
`calculate_dt` is a pretty generic name for CLIMA -- this should probably not be exported.

Reply:
yeah, I'm still figuring out how modules and exporting work in Julia. I can probably remove the export and just add a `CLIMA.` in `simple_box.jl`

Reply:
So I made this a method defined in `DGmethods/balancelaw.jl` that we can use multiple dispatch on with the type of model. I put some examples in `Atmos.jl` and `linear.jl` but these probably need to be updated

Reply:
I added `ode_dt` back, as the ability to override the timestep computation can be useful. Shouldn't affect you.

Reply:
sounds good to me

Reply:
Aaannd I added `Courant_number` back too, after some discussion with the Atmos folks. Again, this shouldn't affect you, but let's chat if needed.